docs/source/glossary.rst

```diff
@@ -0,0 +1,78 @@
+Glossary
+========
+
+.. _Umbral: https://github.com/nucypher/umbral-doc/blob/master/umbral-doc.pdf
+
+.. glossary::
+
+    Alice
+      *"The Data Owner"* :term:`Character` - retains full control over the data encrypted for her and determines whom to share the data with.
+
+    Bob
+      *"The Data Recipient"* :term:`Character` - the data recipient that :term:`Alice` intends to share data with.
+
+    Capsule
+      Encrypted symmetric key (:term:`KEM`) that is eventually re-encrypted.
+
+    cFrag
+      A fragment of ciphertext that is a partial re-encryption produced by a :term:`kFrag` operation on a :term:`Capsule`.
+
+    Character
+      A common term for any entity fulfilling a particular role in our cryptographic narrative.
+
+    DEM
+      Data encapsulation mechanism - data encrypted with a symmetric key.
+
+    Enrico
+      *"The Encryptor"* :term:`Character` - a data source that encrypts data on behalf of :term:`Alice` and produces a :term:`MessageKit`.
+
+    Felix
+      *"The Faucet"* :term:`Character` - provides *testnet* NU tokens for nodes on the test NuCypher Network.
+
+    KEM
+      Key encapsulation mechanism - a symmetric key encrypted with an asymmetric key
```

Comment:
KEM and DEM are "approaches" or "methodologies". They are not objects. I think it's better to just not include them in the glossary.

Reply:
@cygnusv I do reference them as part of the definition for `Capsule`, `MessageKit`. Does that change your mind or no? If not, that's fine - I'll remove the `:term:` references. wdyt?